package.json

```diff
@@ -40,10 +40,11 @@
     "envelope-spec": "^1.1.1",
     "fast-deep-equal": "^3.1.3",
     "is-canonical-base64": "^1.1.1",
+    "jitdb": "^7.0.7",
```

Comment:
had to add this as tests were failing when i pulled in master because of missing jitdb/operators dep. but when installing this i got

```
% pnpm add jitdb
Packages: +1
+
Progress: resolved 521, reused 521, downloaded 0, added 1, done

dependencies:
+ jitdb 7.0.7

 WARN  Issues with peer dependencies found
.
└─┬ jitdb 7.0.7
  └── ✕ missing peer async-append-only-log@^4.3.2
Peer dependencies that should be installed:
  async-append-only-log@^4.3.2

Done in 5.9s
```

things seem to work though

@mixmix